docs/index-extra-title-page.html

```diff
@@ -63,7 +63,7 @@
               >
             </li>
             <li class="listitem">
-              <a href="https://www.elastic.co/guide/en/kibana/current/index-patterns.html"
+              <a href="https://www.elastic.co/guide/en/kibana/8.0/data-views.html"
```

Comment:
Not sure what this change does. Also don't know if 8.0 should be hard-coded.

Reply:
@jrodewig It has to be hard coded for now.  There is no data-views file in 8.0 and beyond.

Reply:
Makes sense. Thanks!